packages/client/tests/functional/0-legacy-ports/malformed-id/tests.ts

```diff
@@ -21,23 +21,27 @@ testMatrix.setupTestSuite(
       if (clientMeta.runtime === 'edge') {
```

Comment:
I think this `if` does not make sense anymore, snapshot is identical in all branches.